test/integration_test.py

```diff
@@ -158,6 +164,12 @@ class IntegrationTestCase(AzulTestCase, metaclass=ABCMeta):
     def azul_client(self):
         return AzulClient()
 
+    def random_prefix(self, length: int) -> str:
```

Comment:
Combined with the refactoring above, this can become a method again.

Reply:
I'm guessing you meant "local method"